changelog.config.js

```diff
@@ -0,0 +1,88 @@
+const COMPONENT_SPECS = require('genesys-spark-components/build/component-specs/component-specs.json');
+const CHART_COMPONENT_SPECS = require('genesys-spark-chart-components/build/component-specs/component-specs.json');
+
+const componentSpecs = Object.assign(COMPONENT_SPECS, CHART_COMPONENT_SPECS);
+
+const getComponentSpec = function () {
+  let scopeList = ['other'];
+  for (const [key] of Object.entries(componentSpecs)) {
+    scopeList.push(key);
+  }
+  return scopeList;
+};
+
+module.exports = {
+  disableEmoji: true,
```

Comment:
should this be `false` if we are using `emoji` in the format ?

Reply:
Do we want to have emoji? The default config file uses emoji but personally I don't like emoji. What do we think